server/pulp/server/db/manage.py

```diff
@@ -180,6 +185,12 @@ def ensure_database_indexes():
         model_class.ensure_indexes()
 
 
+def _get_running_worker_names():
```

Comment:
Can this become part of a custom QuerySet that is attached to the Worker model? That would allow usage like `Worker.get_running_workers()` to be called from anywhere in the codebase.